src/controllers/controller.polarArea.js

```diff
@@ -164,8 +175,8 @@ module.exports = function(Chart) {
 			// var negHalfPI = -0.5 * Math.PI;
 			var datasetStartAngle = opts.startAngle;
 			var distance = arc.hidden ? 0 : scale.getDistanceFromCenterForValue(dataset.data[index]);
-			var startAngle = datasetStartAngle + (circumference * visibleCount);
```

Comment:
Travis complains that `visibleCount` is not anymore used and it seems that the whole block from line 167 to 173 can be removed. Does that make sense?

Reply:
Oh right, thanks "Travis" 😅